packages/node-experimental/src/sdk/client.ts

```diff
@@ -0,0 +1,43 @@
+import type { Tracer } from '@opentelemetry/api';
+import { trace } from '@opentelemetry/api';
+import { NodeClient, SDK_VERSION } from '@sentry/node';
+import { OtelHooks } from '@sentry/opentelemetry-node';
+
+import type { NodePreviewClientOptions } from '../types';
+
+/**
+ * A client built on top of the NodeClient, which provides some otel-specific things on top.
+ */
+export class NodePreviewClient extends NodeClient {
```

Comment:
l: Not sure if I missed something but shouldn't we rather call this `NodeExperimentalClient` to be in line with the package name?

Reply:
Yes! Good catch, I renamed this in the middle of development from node-preview to node-experimental (to better reflect what it is). will rename!